processing/data_collection/gazette/spiders/es_vila_velha.py

```diff
@@ -0,0 +1,56 @@
+import dateparser
+
+from datetime import datetime, date
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+from scrapy import FormRequest, Request
+import os
+
+
+class VilaVelhaSpider(BaseGazetteSpider):
+    name = "es_vila_velha"
+    allowed_domains = ["www.vilavelha.es.gov.br"]
+
+    TERRITORY_ID = "3205200"
+    GAZETTE_URL_CSS = "td:last-child a::attr(href)"
+    GAZETTE_DATE_CSS = "td:nth-child(2) span b::text"
+    GAZETTE_ISSUE_CSS = "td:nth-child(3) span b::text"
+    JAVASCRIPT_POSTBACK_REGEX = r"javascript:__doPostBack\('(.*)',''\)"
+
+    # this is the first date which has gazettes available (edition number 1)
+    start_date = date(2016, 7, 1)
+
+    def start_requests(self):
+        date_param = self.start_date.strftime("%d/%m/%Y")
+        base_url = "https://www.vilavelha.es.gov.br/diariooficial/ConsultaDiario.aspx"
+        gazettes_url = f"{base_url}?dataInicial={date_param}"
+        yield Request(gazettes_url)
+
+    def parse(self, response):
+        for element in response.css("#ctl00_cpConteudo_gvDocumentos tr"):
+            if element.css("th").extract():
```

Comment:
Could you attribute this to a variable with a meaningful name so it gets easier to understand why this type of element is needed to be skipped?